pypuppetdb/api/__init__.py

```diff
@@ -401,6 +401,10 @@ def nodes(self, unreported=2, with_status=False, **kwargs):
                 else:
                     node['status'] = 'unchanged'
 
+                # Override if available
+                if 'latest_report_status' in node:
+                    node['status'] = node['latest_report_status']
```

Comment:
I'm not sure. Can  `latest_report_status` have value `noop`, or must it be computer always?